.github/workflows/linux_build_test.yml

```
@@ -49,7 +49,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
         with:
-          submodules: recursive
+          set-safe-directory: '*'
 
```

I hope this may fix the linux build test failure.

```suggestion

- name: Add exception for dubious ownership
  run: git config --global --add safe.directory /__w/DAGMC/DAGMC
  
```